### DISCUSSION TITLE
Issue with credit card's checkout in Mercado Libre portal with Test users

### DISCUSSION THREAD

**Discussion by td-cristian-torrado** [mercadopago/sdk-js#52] [Q&A] [answered]
Steps to reproduce: 

Preconditions: Have two test account generated by mercado libre sdk

1. Generate a new publish with one account
2. Try to buy this publish using Mercado Pago test credit cards numbers

Expected berhaviour :

The payment is accepted 

Real berhaviour: 

The payment is rejected in master card, visa an Amex  test credit cards. 

This issues is happening since a week, can you please fix it ? thanks

**Comment by lucmantovani** [comment, marked as answer]
Hello @CristianTorrado-TorradoDesarrollos!
I am sorry for the inconvenience in your workflow.

Based on your description, I believe you are building an integration with Mercado Libre.
Even though your are using Mercado Pago's test credit card numbers, I am afraid this issue might not be related to Mercado Pago SDKs.

If you really are building an integration with Mercado Libre, I recommend you to contact support [here](https://developers.mercadolibre.com.ar/devcenter/support).

**Reply by td-cristian-torrado** [reply]
Thanks for the quickly response, i did but i am working the the beta implementation and the support channel in Mercado Libre now is only available for Production registered apps. And this issues is related payments in test environments. And Mercado Libre doesn't have a GitHub channel for developers. Can you bring me another channel to can request the fix into the test environment of ML ?

**Reply by lucmantovani** [reply]
Hello @CristianTorrado-TorradoDesarrollos!

I checked internally and unfortunately the support is really currently available for certified applications.
But we are working on a forum that will be open for everyone to post.

Another team is addressing this issue and I am sure that soon it will be fixed!

**Reply by td-cristian-torrado** [reply]
Thanks @lucmantovani  sure is a shame that not perform support to new developers to integrate, that limit a lot bring new developments with solutions to the community of mercado libre. 

Thanks for report the issue, i bring here updates: the issue continue present in Mercado Libre, we cannot use the test credit card to perform tests. Please we needs a quickly solution for that.  We creates a new couple of test users to tesst with those users and is still not working. 

Thanks in advance.

**Reply by lucmantovani** [reply]
Hello @CristianTorrado-TorradoDesarrollos!

Unfortunately there is not much else I can do to help you with this issue 😕
All we can do now is to wait for the fix.

Have a great day!
